api/config/config.go

```diff
@@ -190,9 +190,9 @@ var defaultConfig = &Config{
 	Environment: "dev",
 	Port:        8080,
 
-	Streams: Streams{},
-	Docs:    Documentations{},
-
+	Streams:      Streams{},
+	Docs:         Documentations{},
+	Applications: []modelsv2.Application{},
```

Comment:
Minor fix - the value of an uninitialised slice is `nil`.  Though this is rare in practice, this was a causing an issue in the local run of the UI, without any apps.

<img width="1105" alt="Screenshot 2023-06-11 at 10 59 34 AM" src="https://github.com/caraml-dev/mlp/assets/23465343/59a5a008-23a9-4f63-b743-1f5e90309b0b">